### ISSUE TITLE
fails to create directories

### ISSUE THREAD

**Issue by jahschwa** [amcoder/curse_client#7]
It appears the scripts do not create directories if they do not exist; for example the modpack I tried to install had subfolders in the mod dir

```
/var/lib/gems/2.3.0/gems/rubyzip-1.2.0/lib/zip/entry.rb:575:in `initialize': No such file or directory @ rb_sysopen - /home/redacted/.local/share/multimc/instances/Thermacraft/minecraft/mods/ic2/EJML-core-0.26.jar (Errno::ENOENT)
```

(Note `/home/redacted/.local/share/multimc/instances/Thermacraft/minecraft/mods/` does exist)

I can individually fix the error by creating an empty `ic2` folder, or I can fix it by manually downloading the curse modpack zip and copying its contents to the target directory.

**Comment by amcoder** [comment]
I am able to reproduce this issue with the newer versions of Direwolf20 1.10.

It looks like some of the zip files do not contain directory entries anymore. 

```
$ unzip -l ~/Downloads/FTBPresentsDirewolf20110-1.0.0-1.10.2.zip
Archive:  /Users/amcoder/Downloads/FTBPresentsDirewolf20110-1.0.0-1.10.2.zip
  Length     Date   Time    Name
 --------    ----   ----    ----
     3205  11-14-16 13:31   manifest.json
        0  11-14-16 13:31   overrides/
        0  11-14-16 13:30   overrides/scripts/
      460  11-14-16 13:30   overrides/scripts/FTBPresentsDirewolf20110.zs
        0  11-14-16 13:30   overrides/resources/
        0  11-14-16 13:30   overrides/resources/mainmenu/
     3422  11-14-16 13:30   overrides/resources/mainmenu/midbutton.png
     3848  11-14-16 13:30   overrides/resources/mainmenu/lang.png
--- snip ---
        0  11-14-16 13:31   overrides/mods/
        0  11-14-16 13:30   overrides/mods/ic2/
   341614  11-14-16 13:30   overrides/mods/ic2/EJML-core-0.26.jar
        0  11-14-16 13:30   overrides/mods/1.10.2/
        0  11-14-16 13:30   overrides/config/
```

```
$ unzip -l ~/Downloads/FTBPresentsDirewolf20110-1.2.2-1.10.2.zip
Archive:  /Users/amcoder/Downloads/FTBPresentsDirewolf20110-1.2.2-1.10.2.zip
  Length     Date   Time    Name
 --------    ----   ----    ----
     4212  12-04-16 09:26   manifest.json
      579  12-04-16 09:26   overrides/scripts/FTBPresentsDirewolf20110.zs
  2325076  12-04-16 09:26   overrides/resources/mainmenu/background.png
     1286  12-04-16 09:26   overrides/resources/mainmenu/button.mp3
     1616  12-04-16 09:26   overrides/resources/mainmenu/clearbutton.png
     3848  12-04-16 09:26   overrides/resources/mainmenu/lang.png
   341614  12-04-16 09:26   overrides/mods/ic2/EJML-core-0.26.jar
--- snip ---
```

I'm not sure if not including directory entries is valid or not, but as far as I can tell, no unzip programs I have tested have an issue unzipping them.

I will change it to create the directory if it doesn't exist when extracting a file.

**Comment by amcoder** [comment]
@TheSchwa Thanks for the bug report! It should be fixed in version 0.1.4.
